flow-typed/box-ui-elements-updates.js

```diff
@@ -0,0 +1,931 @@
+/**
```

Comment:
What is this file used for?

Reply:
I had copied over some changes I made in box-ui-elements.js for my own reference. Deleted it once I pushed to make the PR